cypress/e2e/inventory/item-status-date-updates.cy.js

```diff
@@ -185,7 +183,7 @@ describe('ui-inventory: Item status date updates', () => {
   };
 
   // test is looping
-  it('C9200 Item status date updates (folijet) (prokopovych)', { tags: [DevTeams.folijet, TestTypes.long] }, () => {
+  it('C9200 Item status date updates (folijet) (prokopovych)', () => {
```

Comment:
please add prokopovych team tag

Reply:
This test is looping and I deleted all tags from it.